Sources/Valet/Valet.swift

```diff
@@ -377,10 +386,10 @@ public final class Valet: NSObject {
         case .standard:
             serviceAttribute = Service.standard(with: configuration, identifier: identifier, accessibilityDescription: accessibilityDescription)
         #if os(macOS)
-        case .sharedAccessGroupOverride:
-            serviceAttribute = Service.sharedAccessGroup(with: configuration, explicitlySetIdentifier: identifier, accessibilityDescription: accessibilityDescription)
```

Comment:
This was a bug before (only on the `develop--4.0` branch).

It occurs to me that I need to enable explicitly set identifiers that use shared access groups more thought. Right now you can only name your explicitly set shared access group to use the same identifier as your keychain entitlement, which isn't terribly human-readable. I can tackle this in a follow-up PR, as this is an existing shortcoming.